packages/nx-plugin/src/executors/internal/cli.ts

```diff
@@ -9,6 +9,22 @@ export function createCliCommand(options?: {
   )}`;
 }
 
+export function createCliCommandObject(options?: {
+  args?: Record<string, unknown>;
+  command?: string;
+  bin?: string;
+}): {
+  command: string;
+  args: string[];
+  cwd?: string;
+} {
+  const { bin = '@code-pushup/cli', command, args } = options ?? {};
+  return {
+    command: 'npx',
+    args: [bin, ...objectToCliArgs({ _: command ?? [], ...args })],
+  };
+}
```

Comment:
It doesn't look like like this new function is unit tested.

Reply:
Added 3 unit tests